### ISSUE TITLE
boxen not applying $BOXEN_HOME/facts.d facts

### ISSUE THREAD

**Issue by addigy** [boxen/boxen#179]
I tried adding fact files within $BOXEN_HOME/facts.d directory, and it's not automatically picking them up.  If I manually run the '$BOXEN_HOME/bin/facter --external-dir $BOXEN_HOME/facts.d' it picks up all the fact files properly.

I noticed the following BOXEN ruby script is supposed to override the /etc/facter/facts.d dir, with the $BOXEN_HOME/facts.d.  Are others able to successfully add fact files to $BOXEN_HOME/facts.d?

Thanks,
--Jason Dettbarn

**Comment by rafaelfranca** [comment]
Yeah. Which version of puppet-boxen are you using? This feature only works with 3.8.0+

**Comment by addigy** [comment]
I'm running 3.10.1

Kind Regards,
--Jason Dettbarn
M: 305.799.7215
F: 954.756.7116

**Comment by rafaelfranca** [comment]
Actually `$BOXEN_HOME/facts.d` is not loadable, just `$BOXEN_REPO_DIR/facts.d/`. See https://github.com/boxen/puppet-boxen#facts
